docs/source/infrastructure.rst

```diff
@@ -117,27 +112,39 @@ Once the changes are deployed to stage, you should do some testing to make sure
 Pushing to Production
 ~~~~~~~~~~~~~~~~~~~~~
 
-Pushing live requires CloudOps. For non-urgent pushes, you should begin this procedure a few hours in advance to give CloudOps time to notice and respond. For urgent pushes, file the bug immediately and escalate if no action is taken quickly. Either way, you must follow this procedure to push:
+Pushing the backends live requires some button clicking in Jenkins. For each of ``balrog-admin-production``, ``balrog-production``, and ``balrog-agent-production`` in Jenkins do the following:
```

Comment:
Do we recommend waiting for one to complete before clicking the next...or does the ui force you to?

Reply:
These can and should happen in parallel - I'll clarify this.